Makefile

```diff
@@ -76,8 +77,26 @@ run: translate/dist .server-build
 clean:
 	rm -rf translate/dist .server-build
 
-shell:
-	"${DC}" run --rm server //bin/bash
+.run-container:
+	@container=$$(${DOCKER} ps -q --filter ancestor=local/pontoon | head -n 1); \
+	if [ -z "$$container" ]; then \
+  		echo "Trying to start the container" >&2; \
+		"${DC}" up --detach; \
```

Comment:
This is spinning up an existing container if there isn't one already. 
I've tested the code by manually stopping the container before running it.